codegen/test/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(NOT INSTANCES_ONLY)
     add_test(NAME codegen_test_${BASE_NAME} COMMAND codegen_test_${BASE_NAME})
     message("adding test codegen_test_${BASE_NAME}")
     target_link_libraries(codegen_test_${BASE_NAME} ck_rtc ck_host)
-    target_include_directories(codegen_test_${BASE_NAME} PUBLIC include())
+    target_include_directories(codegen_test_${BASE_NAME} PUBLIC ${CK_ROOT}/codegen/test/include)
```

Comment:
This should not be needed, you already linked ck_rtc which should add the include directories.